packages/snack-content/src/sdks/index.ts

```diff
@@ -7,6 +7,7 @@ const assets = {
 };
 
 const sdks: { [version: string]: SDKSpec } = {
+  // TODO(cedric): drop SDK 48
```

Comment:
Note, not going to drop 48 in this PR. This will be a follow-up, likely next week to also bump SDK 52 to latest again.